content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -77,6 +73,7 @@ export class ContextMenuItem extends React.PureComponent {
     super(props);
     this.onClick = this.onClick.bind(this);
     this.onKeyDown = this.onKeyDown.bind(this);
+    this.onKeyUp = this.onKeyUp.bind(this);
```

Comment:
Do we need tabIndex on ContextMenuItem L60 above?

Reply:
Doesn't look like we do. Removed!